telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/webhook/registration/TelegramWebHookRegistrator.java

```diff
@@ -20,8 +20,8 @@ public TelegramWebHookRegistrator(BotApi botApi) {
     }
 
     public boolean registerWebHook(String url) {
-        log.info("Registering webhook: {}", url);
-        return setWebHook(url);
+        log.info("Registering webhook: {}", url + "/webhook");
```

Comment:
I found "webhook" string three times in code already, I think we should honor it and promote to constant :)